lib/libyui_client/widgets/table.rb

```diff
@@ -4,35 +4,137 @@ module LibyuiClient
+      #               "labels": [
+      #                   "test.item.2",
+      #                   "",
+      #                   "",
+      #                   ""
+      #               ]
+      #           }
+      #       ],
+      #       "items_count": 2
+      #     }
+      # @example
+      #   app.table(id: 'test_id').items
+      #   # [test.item.1, "", "", ""]
+      #   # [test.item.2, "", "", ""]
+      def items
+        property(:items).map { |x| x[:labels] }
+      end
+
       # Sends action to select a row in a table.
       # @param value [String] value to select in the table.
-      # @param column_id [Integer] number of column in zero-based numbering format (i.e. the first column is 0)
+      # @param column [String] column name where value is present
       # @return [Table] in case action is successful
-      # @example Select "test.item.2" in the table
+      # @example Select row with value "test.item.2" for column "header1" in table with id 'test_id'
+      #   app.table(id: 'test_id').select(value: 'test.item.2', column: header1)
+      def select(value:, column: nil)
+        params = { action: Actions::SELECT, value: value }
+        params.merge!(column: get_index(column)) unless column.nil?
+        action(params)
+        self
+      end
+
+      # Returns the list of items currently selected from the table.
+      # @return [Array] array of [Array] objects containing values for each column
+      # @example Get items from the table with id "test_id"
       #     {
       #       "class": "YTable",
-      #       "columns": 1,
-      #       "id": "test_table",
+      #       "columns": 4,
+      #       "header": [
+      #         "header1",
+      #         "header2",
+      #         "header3",
+      #         "header4"
+      #       ],
+      #       "id": "test_id",
       #       "items": [
       #           {
       #               "labels": [
-      #                   "test.item.1"
+      #                   "test.item.1",
+      #                   "",
+      #                   "",
+      #                   ""
       #               ],
       #               "selected": true
       #           },
       #           {
       #               "labels": [
-      #                     "test.item.2"
-      #               ]
+      #                   "test.item.2",
+      #                   "",
+      #                   "",
+      #                   ""
+      #               ],
+      #               "selected": true
       #           }
       #       ],
       #       "items_count": 2
       #     }
       # @example
-      #   app.table(id: 'test_table').select_row(value: 'test.item.2', column_id: 0)
-      def select_row(value:, column_id:)
-        action(action: Actions::SELECT, value: value, column: column_id)
-        self
+      #   app.table(id: 'test_id').selected_items
+      #   # [test.item.1, "", "", ""]
+      #   # [test.item.2, "", "", ""]
+      def selected_items
```

Comment:
Side note here, actually there is a property which shows if table allows multiple rows selection (YUIProperty_MultiSelection = "MultiSelection"). I don't think we need any special handling for it though.
But my question is have you tested it for multiple rows selection? Does it work properly or we need some patches for server side? If not - I will check that, as I'm not 100% sure if that works.